src/eventtimes.jl

```diff
@@ -6,14 +6,14 @@
     EventTime{T}
 
 Immutable object containing the real-valued time to an event as well as an indicator of
-whether the time corresponds to an observed event or right censoring.
+whether the time corresponds to an observed event (`true`) or right censoring (`false`).
 """
 struct EventTime{T<:Real}
```

Comment:
I can't help wondering whether this wouldn't be another case where using `Union{Event{T}, Censored{T}}` wouldn't make sense, with one type for uncensored and one type for censored events. That would probably be more confusing that helpful, but I thought I would mention it. At least the advantage would be that the memory layout of arrays of event times would be efficient (no additional padding to align e.g. 64 + 8 bits).

Reply:
Hm, that's an interesting idea. Having a separate type for right censoring might be a good distinction from left truncation or whatever else should we ever implement those, but I also don't want to be in another `Nullable{T}` situation, where everything requires unwrapping to be useful. (Though I guess `EventTime` is like that now...)